sympy/polys/tests/test_polytools.py

```diff
@@ -3933,3 +3933,38 @@ def test_issue_20985():
     w, R = symbols('w R')
     poly = Poly(1.0 + I*w/R, w, 1/R)
     assert poly.degree() == S(1)
+
+
+def test_factor_system():
+    from sympy.polys.polytools import factor_system
+    from sympy import symbols, sqrt
+
+    x, y = symbols('x y')
```

Comment:
These imports should go to the top of the file.

There should be tests for the other functions, not just `factor_system`.